helm-chart/environments/minikube.yaml

```
@@ -1,3 +1,17 @@
 imagePullSecrets: 
   - name: ghcr-registry-auth
-  
\ No newline at end of file
+
+config:
+  # This value should be in decimal, not hexadecimal format.
+  CHAIN_ID: ""
+  MIRROR_NODE_URL: ""
+  LOG_LEVEL: ""
+  local:
+    HEDERA_NETWORK: {"192.168.1.10:50211":"0.0.3","192.168.1.11:50211":"0.0.4"}
```

Should be local node version 
```suggestion
    HEDERA_NETWORK: {"127.0.0.1:50211":"0.0.3"}
```

---

updated